website/blog/2024-02-29-cortex-slack.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "LLM-powered Analytics Engineering: How we're using AI inside of our dbt project, today, with no new tools."
+description: "With Snowflake's new Cortex functions and the dbt Cloud orchestrator, we can quickly and safely do analytics that was once impractical."
+slug: dbt-models-with-snowflake-cortex
+
+authors: [joel_labes]
+
+tags: [analytics craft, data ecosystem]
+hide_table_of_contents: false
+
+date: 2024-03-04
+is_featured: true
+---
+
+## Cloud Data Platforms make new things possible; dbt helps you put them into production
+
+The original paradigm shift that enabled dbt to exist and be useful was databases going to the cloud.
+
+All of a sudden it was possible for more people to do better data work as huge blockers became huge opportunities:
+
+- We could now dynamically scale compute on-demand, without upgrading to a larger on-prem database.
+- We could now store and query enormous datasets like clickstream data, without pre-aggregating and transforming it.
+
+Today, the next wave of innovation is happening in AI and LLMs, and it's coming to the cloud data platforms dbt practitioners are already using every day. For one example, Snowflake have just released their [Cortex functions](https://docs.snowflake.com/LIMITEDACCESS/cortex-functions) to access LLM-powered tools tuned for running common tasks against your existing datasets. In doing so, there are a new set of opportunities available to us:
+
+<!-- truncate -->
+
+- We can now **derive meaning from large unstructured blocks of text**, without painstakingly building complex regexes
+- We can now **summarize or translate content** without having to call out to external third-party APIs.
+- Most significantly, we can now **bake reasoning capabilities into our dbt models** by describing what we want to happen.
+
+Analytics Engineers have always existed at the intersection of business context and data - LLMs on the warehouse make it possible to embed more business context _and_ unlock more data, increasing our leverage in both directions at once.
+
+## Anatomy of an LLM-powered workflow
+
+When we were looking for a first AI-powered use case in our analytics stack, we wanted to find something that:
```

Comment:
I think it's meaningful to call out the fact that this is our first developer blog post, on official dbt documentation, about AI. Could we speak a little about that and call that out? I think it's 1. rad that this is happening/celebrate it 2. really emphasize that this is new/our first jump in

Reply:
It also flows more seamlessly because I actually don't understand the why are you looking for our first AI powered use case

Reply:
Done!